src/common/common.js

```diff
@@ -129,7 +129,8 @@ function cleanCommentData(comment, skipAdditional = []) {
   ];
   const toSkip = [...skipDefaults, ...skipAdditional];
 
-  comment = comment.replace(/<span[^>]*>(.*?)<\/span>/, "$1")
+  //https://regex101.com/r/aAZJcy/2
+  comment = comment.replace(/(?<!```.*)<span[^>]*>(.*)<\/span>|<span[^>]*>(.*)<\/span>(?!.*```)/g, "$1")
```

Comment:
Whats' the rational for matching something that starts but not ends with a code block and ends but doesn't start with a code block?

Reply:
This code removes `<span>` tags and replaces with the contents.  The idea here is that we want to keep the raw string if we're inside a code block.

Reply:
I see the negative look ahead, I'm wondering what the reason to match the test lines `"```<span>should match</span>"` and `"<span>should match</span>```"` are. Those two seem like they symbolize the start of a multiline code block so I'm wondering why they are also included to be replaced instead of being left alone.

Reply:
The current approach doesn't allow for multi-line code blocks.  The previous approach didn't respect code blocks at all; fixing that revealed a single case in our code where a `<span>` was embedded in a single-line code block (and it wasn't stripped/replaced because it was the second tag in the comment).  If we want to support embedded `<span>` tags in multi-line code blocks then a more comprehensive change may be needed.